Mobile header and healine

t-bello7 opened the pull request:
## What
- Mobile View Header Design
- set up the project with linter configurations
- add utility and reset CSS
- add header and headline in the HTML file
- style header and headline in the style.css file
## Why 
- Designing a layout with modern CSS flexbox for better accessibility

t-bello7 committed:
build: setup rep professionally
- added eslint,hint and stylelint
- add github workflows
- add README file

t-bello7 committed:
fea: Add html boilertemplate
- add style, reset and utility css file
- add html5 boilertemplate to html5

t-bello7 committed:
style: add imgs in header

t-bello7 committed:
style: add the elements for the home page
- add all the elements for the home page
- position the elements

t-bello7 committed:
build: add overknow to gitignore file

t-bello7 committed:
fix: change contact link button to list

t-bello7 committed:
fix: remove extra paddings and margins on ul

t-bello7 committed:
build: removed eslint run job

t-bello7 committed:
fix: headbar
- change menu icon from font awesome icon to figma file icon
- change color text to white
- add alt text for menu icon

t-bello7 committed:
refactor: index.html
- add google fonts for text
- add margin flow for elements in headline section

t-bello7 committed:
Merge branch 'headline' into btn-icon

t-bello7 committed:
fix: correct stylesheet errors

danmainah commented on .github/workflows/.linter.yml:
```diff
@@ -0,0 +1,64 @@
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  # eslint:
+  #   name: ESLint
+  #   runs-on: ubuntu-18.04
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v1
+  #       with:
+  #         node-version: "12.x"
+  #     - name: Setup ESLint
+  #       run: |
+  #         npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+  #         [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+  #     - name: ESLint Report
+  #       run: npx eslint .
```
- [x] Kindly remove the commented code in your file, its note good practice to have commented code in your codebase, also its bad practice to edit your linter files.

danmainah commented on index.html:
```diff
@@ -0,0 +1,32 @@
+<body>
+    <header class="container bg-gray">
+        <div class="flex header">
+            <span class="text-light">John Doe</span>
```
- [x] Kindly ensure your logo is a link using an `<a>` tag, this is a best practice as logos are used as links to the index page of the website.

danmainah commented on index.html:
```diff
@@ -0,0 +1,32 @@
+    <header class="container bg-gray">
+        <div class="flex header">
+            <span class="text-light">John Doe</span>
+            <button class="btn-icon" type="button"> <img src="assets/img/menu.png" alt="menu"></button>
```
- [x] Please ensure your hamburger button has a `<nav>` tag, this is because in upcoming milestones your hamburger will be a menu containing nav items.

danmainah commented on css/utility.css:
```diff
@@ -0,0 +1,51 @@
+  margin: 0 auto;
+}
+
+/* .flow  > * {
+
+} */
```
- [x] Kindly remove this commented code, it's always good practice to remove commented code in your codebase.

danmainah commented on index.html:
```diff
@@ -0,0 +1,32 @@
+        <span class="text-primary-2 ff-roboto mg-1 ">I'm a Software Developer</span>
+        <p class="text-light line-height-2 ff-poppins mg-1">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestitate to contact me.</p>
+        <ul class="flex contact-link mg-1"> 
+           <li> <a href="https://github.com/t-bello7" class="btn-contact"><img src="assets/img/github.png" alt=""> </a>  </li>
```
- [x] Kindly ensure you have a proper background image, this is achieved using a stylesheet adding an image as background [check example here](https://www.w3schools.com/cssref/pr_background-image.asp)

danmainah reviewed (changes requested):
Hi @t-bello7 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [x] Kindly ensure your pull request description is descriptive, currently, its too short and does not explain all changes made in this milestone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

t-bello7 committed:
refactor:
- change icons in a tag to backround image
- removed unnecessary comments from linter and style.css file
- add resources to readme file
- add nav menu to header
- fix stylelint issues

t-bello7 committed:
refactor:
- change icons in a tag to backround image
- removed unnecessary comments from linter and style.css file
- add resources to readme file
- add nav menu to header
- fix stylelint issues

OLIPLICHE commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    <header class="container bg-gray">
+        <div class="flex header">
+            <a href="#" class="text-light">John Doe</a>
+            <button class="btn-icon" type="button"> <img src="assets/img/menu.png" alt="menu"></button>
```
- [x] Please nest the #hamburger menu# button withing `<nav>` tag. This allows screen reader users to navigate directly to the navigation landmark. Kindly follow the requirement from the previous review.

OLIPLICHE commented on index.html:
```diff
@@ -0,0 +1,33 @@
+        <div class="flex header">
+            <a href="#" class="text-light">John Doe</a>
+            <button class="btn-icon" type="button"> <img src="assets/img/menu.png" alt="menu"></button>
+            <nav></nav>
```
- [x] Please this is not what the previous review requested. Kindly delete it to make your code clean. Apply what I requested above

OLIPLICHE commented on css/style.css:
```diff
@@ -0,0 +1,51 @@
+  align-items: center;
+}
+
+.btn-icon {
+  background: transparent;
+  border: none;
+}
+
+.home {
```
- [x] Please you did not fix what the previous review noticed. 
- See your design below: 
- 
![ok](https://user-images.githubusercontent.com/63972670/164555267-e183da65-6fc0-43a2-a6a1-623aa836dff5.PNG)

- [x] I suggest that you use "padding 3rem 2rem" or something similar to fix the home section. " padding-left: 2rem" for the header section or something else which can fix it.
- [x] please push the "hamburger" button to the right side. "margin or padding" can also fix it.
- [x] Here is the link to the [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

OLIPLICHE reviewed (changes requested):
Hi @t-bello7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is well defined 👏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

t-bello7 committed:
Fix :merge issues from deleted repo-setup remote

t-bello7 committed:
Fix :merge issues from deleted repo-setup remote

t-bello7 committed:
Merge branch 'mob-header' of https://github.com/t-bello7/micro-portfolio into mob-header

t-bello7 committed:
Merge branch 'mob-header' of https://github.com/t-bello7/micro-portfolio into mob-header

t-bello7 committed:
Merge branch 'mob-header' of https://github.com/t-bello7/micro-portfolio into mob-header

t-bello7 committed:
fix: stylelint error

KDlamini reviewed (approved):
### Hi @t-bello7 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the project requirements. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 
![oscars-standing-ovation](https://user-images.githubusercontent.com/38088818/164718535-96bbc886-6556-45a8-81c5-92cbe12daf5c.gif)

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
